### PR TITLE
cp-150: Renamed chat list to chat

### PR DESCRIPTION
**Pull request by iamAlinaaa**
Successfully renamed 'Chat list' to 'Chat'

Снимок экрана 2023-09-03 142512
Снимок экрана 2023-09-03 142529

### CODE REVIEW

**Review by dmytro-kucherenko — DISMISSED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]

File: mobile/src/libs/enums/navigation/main-screen-name.enum.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const MainScreenName = {
   HOME: 'Home',
-  CHAT_LIST: 'Chat list',
+  CHAT_LIST: 'Chat',
```
Comment (liza-veis) [inline mobile/src/libs/enums/navigation/main-screen-name.enum.ts]:
```suggestion
  CHAT: 'Chat',
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: mobile/src/libs/enums/navigation/main-screen-name.enum.ts (hunk 1 above)
Comment (iamAlinaaa) [reply mobile/src/libs/enums/navigation/main-screen-name.enum.ts]: @Liza-Veis there will be separate screen named CHAT from #60 , thats why i left CHAT_LIST 
Снимок экрана 2023-09-03 212948

Comment (liza-veis) [reply mobile/src/libs/enums/navigation/main-screen-name.enum.ts]: It's okay if we have screens with the same name in different navigations. Even if it was a problem, it doesn't matter how we name the keys of the enum, what matters is how we name the values of the enum

Comment (iamAlinaaa) [reply mobile/src/libs/enums/navigation/main-screen-name.enum.ts]: @Liza-Veis renamed CHAT_LIST to CHAT

**Review by dmytro-kucherenko — DISMISSED** [review]

**Review by fshabanov — DISMISSED** [review]

**Review by v0ldemar01 — DISMISSED** [review]

**Review by Vika030718 — DISMISSED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by OlesiaTr — DISMISSED** [review]

**Review by liza-veis — DISMISSED** [review]

**Review by fshabanov — DISMISSED** [review]

**Review by Nanaier — DISMISSED** [review]

**Review by dmytro-kucherenko — APPROVED** [review]

**Review by OlesiaTr — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**